### ISSUE TITLE
error[E0432]: unresolved import `std::backtrace` when building with rustc 1.38.0-dev

### ISSUE THREAD

**Issue by strohel** [dtolnay/anyhow#20]
Hi,
I get following errors if I try to compile anyhow 1.0.12 with rustc 1.38.0 itself compiled in `dev` channel:
```bash
error[E0432]: unresolved import `std::backtrace`
 --> src/backtrace.rs:2:21
  |
2 | pub(crate) use std::backtrace::Backtrace;
  |                     ^^^^^^^^^ could not find `backtrace` in `std`

error[E0432]: unresolved import `std::backtrace`
 --> src/context.rs:7:10
  |
7 | use std::backtrace::Backtrace;
  |          ^^^^^^^^^ could not find `backtrace` in `std`
```
```bash
strohel@edgy ~/projekty/anyhow $ rustc --version
rustc 1.38.0-dev
```

It seems to be caused by the fact that `cargo:rustc-cfg=backtrace` is errorneously enabled in `build.rs`. If I do a simple change
```diff
diff --git a/build.rs b/build.rs
index dcc19c6..65541b4 100644
--- a/build.rs
+++ b/build.rs
@@ -34,6 +34,6 @@ fn rustc_version() -> Option<Compiler> {
     };
 
     Some(Compiler {
-        nightly: version.contains("nightly") || version.contains("dev"),
+        nightly: version.contains("nightly"),
     })
 }
```
then the crate compiles and tests pass (but that probably isn't the right thing to do).

Perhaps presence of `std::backtrace` can be tested more directly (maybe even without `build.rs`?).

**Comment by dtolnay** [comment]
Thanks! Should be fixed in anyhow 1.0.13.

**Comment by strohel** [comment]
Works indeed, thanks a lot, that was light-speed.
